MyHeartCounts/MyHeartCountsDelegate.swift

```diff
@@ -31,6 +31,7 @@ class MyHeartCountsDelegate: SpeziAppDelegate { // swiftlint:disable:this file_t
             ConfigureFirebaseAppAccessor()
             BulkHealthExporter()
             HistoricalHealthSamplesExportManager()
+            StudyDefinitionLoader.shared
```

Comment:
What is the need for this singleton; couldn't we rely on the Spezi Module injection mechanism here?